### PR TITLE
[Surrey] Cobrand front-end

### PR DESCRIPTION
**Pull request by lucascumsille**
Fixes: https://github.com/mysociety/societyworks/issues/4228

Preview:

### Emails
Screenshot 2024-04-01 at 10 01 20
Screenshot 2024-04-01 at 10 01 08

### Desktop
Screenshot 2024-04-01 at 10 03 22
Screenshot 2024-04-01 at 10 02 21
Screenshot 2024-04-01 at 10 02 12
Screenshot 2024-04-01 at 10 01 55
Screenshot 2024-04-01 at 10 01 48
Screenshot 2024-04-01 at 10 27 45

### Mobile
https://github.com/mysociety/fixmystreet/assets/13790153/8b191275-5f44-4ed2-98b0-6083b11bc43d

### CODE REVIEW

File: templates/web/surrey/footer_extra.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            <div class="col-12">
+                <ul class="scc-footer__list">
+                    <li class="scc-footer__item">
+                        <a class="scc-footer__link" href="https://www.surreycc.gov.uk/"><span class="sr-only">Surrey County Council </span>Home</a>
+                    </li>
+                    <li class="scc-footer__item">
+                        © Surrey County Council 2024
+                    </li>
+                    
+                        <li class="scc-footer__item">
```
Comment (davea) [inline templates/web/surrey/footer_extra.html:89]: I notice this exists on their own site, but does it make sense for it to be included on FMS?

**Review by davea — APPROVED** [review]
Just one little thing about `ID: 94` but otherwise 👍

**Review by davea — APPROVED** [review]
🎉

**Comment by lucascumsille** [comment]
Thanks, @davea. I just erased that bit; yeah, I wasn't sure if it was necessary, but I kept it for the sake of making a replica. but I don't think anyone is going to miss it.

**Pull request merged**